share/smack/svcomp/utils.py

```diff
@@ -423,6 +439,7 @@ def verify_bpl_svcomp(args):
     sys.exit(smack.top.results(args)[result])
 
 def write_error_file(args, status, verifier_output):
+  return
```

Comment:
This seems really hacky. What is happening here?

Reply:
The C error traces generated by the updated Corral version cannot be correctly converted into SV-COMP witness format. So I just disabled the witness generation.